### PR TITLE
Pipeline health - check that master is last release

### PR DESCRIPTION
**Pull request by ewels**
* Compare commit hash of the last release against the commit hash on master
* Restyle invalid tests to be green instead of grey, to improve contrast of failing tests

### CODE REVIEW

**Comment by ewels** [comment]
Closes #639

File: public_html/pipeline_health.php [hunk 1]
```diff
@@ -529,13 +529,14 @@ private function _save_cache_data($path, $data, $encode_json=true){
...
     $test_url = str_replace('{repo}', $this->name, $test_url);
+    $test_url = str_replace('{latest-tag}', $this->last_release->tag_name, $test_url);
     if(is_null($this->$test_name)){
       echo '<td class="table-secondary text-center" title="<strong>'.$this->name.':</strong> '.$this->test_descriptions[$test_name].'" data-toggle="tooltip" data-html="true">
         <a href="'.$test_url.'" class="d-block" target="_blank"><i class="fas fa-question text-secondary"></i></a>
       </td>';
     } else if($this->$test_name === -1){
-      echo '<td class="table-secondary text-center" title="<strong>'.$this->name.':</strong> '.$this->test_descriptions[$test_name].'" data-toggle="tooltip" data-html="true">
-        <a href="'.$test_url.'" class="d-block" target="_blank"><i class="fas fa-times text-secondary"></i></a>
+      echo '<td class="table-success text-center" title="<strong>'.$this->name.':</strong> '.$this->test_descriptions[$test_name].'" data-toggle="tooltip" data-html="true">
```
Comment (mashehu) [inline public_html/pipeline_health.php:538]:
```suggestion
      echo '<td class="table-warning text-center" title="<strong>'.$this->name.':</strong> '.$this->test_descriptions[$test_name].'" data-toggle="tooltip" data-html="true">
```
green = good, yellow = 🤷‍♂️, imo.

**Review by mashehu — APPROVED** [review]

File: public_html/pipeline_health.php (hunk 1 above)
Comment (ewels) [reply public_html/pipeline_health.php:538]: Yeah that's what I originally had, but then I changed it to grey a while backto be more of an `N/A` than 🤷🏻‍♂️ 

But now even the grey I find a bit tricky as you have a wall of green then one column that is all mixed grey and red. We generally don't care about green or grey and only care about red, so switching it to green like this makes the reds pop out much more.

Comment (ewels) [reply public_html/pipeline_health.php:538]: Take a look after merge - can always change it back if it bothers you.. 😄

Comment (mashehu) [reply public_html/pipeline_health.php:538]: okay, makes sense to have a red and not-red status. 👍

**Pull request merged**